lifemonitor/api/models/rocrate.py

```diff
@@ -115,6 +114,10 @@ def load_metadata(self) -> dict:
         raise lm_exceptions.NotAuthorizedException(detail=f"Not authorized to download {self.uri}", original_errors=errors)
 
     def download(self, target_path: str) -> str:
+        # report if the workflow is not longer available on the origin server
+        if self._metadata and not check_resource_exists(self.uri):
+            raise lm_exceptions.DownloadException(detail=f"Workflow not found on the origin: {self.uri}", status=410)
```

Comment:
I'd change the message to just `"Not found: {self.uri}"`. It's clearer and more concise.

Reply:
Fixed in https://github.com/crs4/life_monitor/pull/146/commits/f721ad65084e236591c33b2dadf24b8c1c1b6466